share/spice/just_delete_me/just_delete_me.js

```diff
@@ -0,0 +1,67 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_just_delete_me = function(api_result) {
+        if (!api_result || api_result.length < 1) {
+            return Spice.failed('just_delete_me');
+        }
+        var script = $('[src*="/js/spice/just_delete_me/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/just_delete_me\/([^\/]+)/)[1],
+            decodedQuery = decodeURIComponent(query).split(" ")[0].toLowerCase();
+
+        if (!decodedQuery || decodedQuery.length < 4) {
+            return Spice.failed('just_delete_me');
+        }
+
+        var exact = api_result.find(function(item) { return item.name.toLowerCase() === decodedQuery}) ||
```

Comment:
It looks like `array.find()` isn't supported by IE, Opera, etc. We need to use methods that work across all browsers.

I would suggest you write this as a loop: `$.foreach(api_result, function(item) {...});`